### PR TITLE
Dev_jy: sell.html and sell-info.html

### PR DESCRIPTION
**Pull request by cheesecat47**
Fix sell.html and sell-info.html pages.

sell.html
- Now you can see the list of items.
- There are left things to handle with buyer DB.

sell-info.html
- If you click a specific item from sell list, the sell-info page shows the detail of that item.

### CODE REVIEW

File: dev/app/src/index.js [hunk 1]
```diff
@@ -275,6 +281,19 @@ const App = {
...
       $("#refund-count").html(i[5]);
     }
+  },
+
+  renderSellInfo: async function (productId) {
+    const {
+      getProduct,
+      productIndex
+    } = this.instance.methods;
+    var p = await getProduct(productId).call();
```
Comment (micro-xl) [inline dev/app/src/index.js:291]: Get product from DB, not Blockchain.
We have an API about it. You can check it in the readme.md file on master branch

File: dev/app/src/sell-info.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  <!-- Additional CSS Files -->
+  <link rel="stylesheet" href="assets/css/fontawesome.css">
+  <link rel="stylesheet" href="assets/css/tooplate-main.css">
+  <link rel="stylesheet" href="assets/css/owl.css">
+  <link rel="stylesheet" href="assets/css/flex-slider.css">
+  <link rel="stylesheet" href="assets/css/devjy.css">
+
+  <script type="text/javascript">
+    function display_alert() {
+      alert("OK!!")
```
Comment (micro-xl) [inline dev/app/src/sell-info.html:27]: add "발송완료! 구매자가 배송확인시 이더가 지급됩니다." statement on alert

**Comment by micro-xl** [comment]
Little modification required

File: dev/app/src/index.js (hunk 1 above)
Comment (cheesecat47) [reply dev/app/src/index.js:291]: There's an issue.
Currently, blockchainId is defined as an integer thus query API can't return the right value.
If I request '?blockchainId=1', I think the API recognizes 1 as a String value.

**Pull request closed without merging**